test/web-platform-tests/to-run.yaml

```diff
@@ -81,6 +81,41 @@ window-screen-width.html: [fail, Test not applicable - no output device]
 
 ---
 
+DIR: custom-elements
+
+CustomElementRegistry.html: [fail, Promise identity discontinuity,
+                                   webidl2js doesn't deal well with tests using Proxies to verify properties access]
```

Comment:
Can you say more about these failures? webidl2js strives to work well in these ways, so fixing it should not be too hard.

Reply:
The test wraps the custom element constructor into a Proxy and checks all the property access on it (`constructor`, `prototype`, `attributeChangedCallback`, ...). Because webidl2js uses a Symbol to like the wrapper to the implementation, the Proxy catches the Symbol property access making the test fail.

Reply:
**WebIDL2JS**&nbsp;should&nbsp;use a&nbsp;**WeakMap** to&nbsp;map&nbsp;wrappers to&nbsp;implementations.

Reply:
I don't it is the case: https://github.com/jsdom/webidl2js/blob/master/lib/output/utils.js#L34-L36

Reply:
I’ve&nbsp;created&nbsp;<https://github.com/jsdom/webidl2js/pull/155> to&nbsp;update&nbsp;**WebIDL2JS** to&nbsp;use&nbsp;**WeakMap**s for&nbsp;the&nbsp;wrapper&nbsp;→&nbsp;impl&nbsp;mapping.